src/Rings/ReesAlgebra.jl

```diff
@@ -0,0 +1,185 @@
+########################################################################
+# Rees algebras of modules 
+#
+# Following Eisenbud, Huneke, Ulrich: "What is the Rees algebra 
+# of a module?"; arXiv:math/0209187v1
+########################################################################
+
+@Markdown.doc """
+    rees_algebra(f::ModuleFPHom{<:ModuleFP, <:FreeMod}; check::Bool=true)
+
+For a *versal* (†) morphism ``f : M → F`` of a module ``M`` into a free 
+module ``F`` this computes the Rees algebra of ``M`` according to [1].
+
+!!!Note: if `check` is set to `true`, the method will check the sufficient
```

Comment:
I don't think this is valid admonition syntax; it must be `!!! note` on a line of its own, the content then follows with indentation, see <https://docs.julialang.org/en/v1.8/stdlib/Markdown/#Admonitions> resp. how we use it elsewhere